### DISCUSSION TITLE
Issue signing into GitHub Enterprise on GitHub desktop

### DISCUSSION THREAD

**Discussion by Kourtney-Burger** [nmfs-ost/PAM-OpenScience#10] [Q&A] [answered]
@rebecca-vanhoeck @shannonrankin 

Hi Becca and Shannon, 

I was just thinking some more about some of the issues we had yesterday with GitHub desktop and signing into enterprise. I was adding some extra directions about that to our seaside chat notes and made the grave mistake of signing out of enterprise on my computer. I am now stuck in the loop of trying to sign on in github desktop and not getting anywhere. I even got a notice that my account was flagged for too many authentication attempts! I am able to add my account without enterprise authentication to GitHub desktop but I cannot make any changes to repos hosted on our orgs. 

I'm really not sure what to do with this situation, I was trying to be proactive to avoid this with tomorrow's meeting but now I've messed up my account. Any advice? Becca, do you have a SWFSC GitHub person I can contact?

**Comment by shannonrankin** [comment]
Look at you, creating learning opportunities so you can look like a Pro
when this happens to some of the newbies 😂

Hopefully Becca has a contact-- otherwise, (or in addition), I would put in
a helpdesk request.

Doh!

**Reply by Kourtney-Burger** [reply]
Haha, I tried! I'll put in a ticket and see where I can get

**Reply by Kourtney-Burger** [reply]
Also, pretty cool this Q&A type of discussion works like stack exchange where I can select an answer

**Reply by shannonrankin** [reply]
Right... just make sure to put your answer (and check it!) once you have a solution!!!  :)

**Reply by rebecca-vanhoeck** [reply]
Oh no, always a bummer when trying to be helpful backfires. Here are the contacts I have for SWFSC GitHub:
Thanh Vu - NOAA Federal <thanh.vu@noaa.gov>,Kevin Stierhoff - NOAA Federal <kevin.stierhoff@noaa.gov>,Lynn Dewitt - NOAA Federal <lynn.dewitt@noaa.gov>

Thanh is the one who has previously replied to my emails.

**Reply by Kourtney-Burger** [reply]
Thanks Becca! I tried to push changes to the lab manual template this morning and it let me do that, even though I was not signed into the enterprise. Maybe that because I am an admin or something, I don't know but I will let you all know what I can find!

**Comment by Kourtney-Burger** [comment, marked as answer]
Based on feedback from the NMFS GitHub team, you do not need to sign onto the enterprise in GitHub desktop, just make sure you are signed into your regular NOAA GitHub account. That blue button is for an on-premise enterprise, while ours is hosted on the cloud. If you are signed into your account on the desktop and still cannot push changes, make sure you are signed onto the [NOAA NMFS enterprise online](https://github.com/enterprises/noaa-nmfs). 

![image](https://github.com/user-attachments/assets/fbcab9d3-1a04-4b89-9bef-9c699fb402ba)

There seems to be some sort of time-out for this, so it is good practice to sign on online first, using your SSO or 2FA (whichever way you have set up) and then try pushing changes using GitHub desktop. Please let me know if you have any questions about this or if you are still running into problems making changes to your repository.
